changelog/_unreleased/2020-09-17-fix-order-listing-sort-by-customer-name.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Fix order listing sort by customer name
+issue: NEXT-9716
+author: belendel
```

Comment:
Can you please use your real name here?
E.g.:
```suggestion
author: Ben
```

Reply:
Done!